app/commands/user/reset_account.rb

```diff
@@ -20,7 +20,7 @@ def call
     )
 
     user.avatar&.destroy
-    User::RemoveRoles.(user, user.roles)
+    User::RemoveRoles.defer(user, user.roles.to_a)
```

Comment:
The `to_a` is needed as `roles` is a `Set`, which sidekiq doesn't support

Reply:
Good catch.